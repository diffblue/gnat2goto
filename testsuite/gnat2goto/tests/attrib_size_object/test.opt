ALL XFAIL gnat2goto does not handle size attribute clauses applied to an object

