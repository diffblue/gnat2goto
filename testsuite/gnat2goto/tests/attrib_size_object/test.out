[object_size.assertion.1] line 15 assertion Small_Pos = 7: SUCCESS
[object_size.assertion.2] line 22 assertion Small_Pos = 8: SUCCESS
[object_size.assertion.4] line 29 Ada Check assertion: SUCCESS
[object_size.assertion.3] line 30 assertion Uncons_SP = 32: SUCCESS
VERIFICATION SUCCESSFUL
